bitseq/sequence_test.go

```diff
@@ -939,9 +941,6 @@ func TestAllocateRandomDeallocate(t *testing.T) {
 	if hnd.Unselected() != uint64(numBits/2) {
 		t.Fatalf("Expected half sequence. Instead found %d free bits.\nSeed: %d\n%s", hnd.unselected, seed, hnd)
 	}
-	if !hnd.head.equal(expected) {
```

Comment:
Allocation not be part of the head. So this verification will not hold true with new change.